apps/rush-lib/src/cli/utilities/InstallManager.ts

```diff
@@ -555,10 +568,11 @@ export default class InstallManager {
       } else {
         // NO: Do an incremental install in the "node_modules" folder
 
-        console.log(`Running "npm prune" in ${this._rushConfiguration.commonTempFolder}`);
-        const npmArgs: string[] = ['prune'];
-        this.pushConfigurationNpmArgs(npmArgs);
-        Utilities.executeCommandWithRetry(npmToolFilename, npmArgs, MAX_INSTALL_ATTEMPTS,
+        console.log(`Running "${this._rushConfiguration.packageManager} prune"`
+          + ` in ${this._rushConfiguration.commonTempFolder}`);
+        const args: string[] = ['prune'];
```

Comment:
>const args: string[] = ['prune']; [](start = 8, length = 33)

Why would PNPM need a "prune" command?

Reply:
Good point

---
In reply to: [151786882](https://github.com/Microsoft/web-build-tools/pull/427#discussion_r151786882) [](ancestors = 151786882)

Reply:
We tested this and it looks like prune is redundant for PNPM

---
In reply to: [152092686](https://github.com/Microsoft/web-build-tools/pull/427#discussion_r152092686) [](ancestors = 152092686,151786882)